docs/release-notes/2.40.1.md

```diff
@@ -0,0 +1,29 @@
+# Percona Monitoring and Management 2.40.1
+
+| **Release date:** | Oct 18, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issues
+
+- [PMM-12592](https://jira.percona.com/browse/PMM-12592) - Fixed an issue where the **Library Panels** for the PMM dashboard were not working.
+- [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading to PMM version 2.40.0, changing the Admin user's password from the terminal was not functioning. The issue has been resolved now.
+- [PMM-12587](https://jira.percona.com/browse/PMM-12587) - During the upgrade to PMM 2.40.0, the process failed with `500 internal server error` and an indefinite loop of `pmm-update-perform-init service` because of incorrect mappings between dashboards, folders, users, and groups. The issue has been resolved.
```

Comment:
```suggestion
- [PMM-12587](https://jira.percona.com/browse/PMM-12587) - 
After the upgrade to 2.40.0  (can go successfully or with `500 internal server error` ), the User can face incorrect mappings between dashboards, folders, users, and groups compared to previous configurations. The issue has been resolved.
```

Reply:
The main problem is broken mapping; all other information is secondary.

Reply:
fixed.